ruby/loops.md

```diff
@@ -0,0 +1,38 @@
+# Loops
+Now that we have gotten our hands a little dirty with ruby, let's discuss the
+concept of loops. Loops allow a program to execute particular tasks as
+many times as needed to achieve a particular goal. This is important because
+thousands of steps within a program can be completed within seconds, increasing
```

Comment:
> This is important because thousands of steps within a program can be completed within seconds, increasing
 productivity and reducing the time and effort that would be required if a user had to do it manually.

This is of course completely correct @calandraz, I think a beginner will have a bit of difficulty being able to relate to this or visualise it. What about describing a couple of common scenarios where loops are useful.
For example.

Looping until a certain condition is met, eg keep looping through code that wants certain input from the user like "yes" or "no" the program will keep looping if anything else is inputted. 

To execute code a certain amount of times eg. when you need print a list of names to the screen exactly 5 times. 

What do you think?